Update bj.m3u tz.m3u mm.m3u

daveesplana opened the pull request:
The stream wont play without a user agent. - (ORTBTV.bj)

daveesplana committed:
Update M3U playlist with new entries and formats

daveesplana committed:
Fix formatting and restore M3U header

BellezaEmporium committed:
Update bj.m3u

BellezaEmporium reviewed (commented):
ORTB 2 https://srtblive.streamakaci.tv/ortb/ortb2-multi/playlist_dvr.m3u8

However ORTB 1 seems to be geo-blocked.

daveesplana committed:
Reformat tz.m3u with additional channels

daveesplana committed:
Update mm.m3u with new channels and streams

daveesplana committed:
Fix M3U formatting and add missing header

daveesplana committed:
Merge branch 'iptv-org:master' into master

daveesplana committed:
Restore M3U playlist with TV channels

Re-added the #EXTM3U header and restored all TV channel entries in the M3U playlist.

Archrootsda committed:
Update tz.m3u
